setup.sh

```diff
@@ -1,22 +1,26 @@
-#!/bin/bash
-Windows=0;
+#! /usr/bin/env bash
 
-while [[ "$#" -gt 0 ]]; do case $1 in
-  --windows) 
-    Windows=1;;
-esac; shift; done
+# Check which Python versions are available
+python --version                                                                                                                                                                                             python --version
```

Comment:
If there's a risk that python isn't installed at all, we could either:
* Explicitly check if the command exists
* Or rely on an installation that is idempotent, and won't be an issue if it's already installed